packages/node_modules/@webex/test-users/src/index.js

```diff
@@ -187,7 +187,9 @@ export function createTestUser(options = {}) {
       'webExSquared'
     ],
     scopes: options.scope || process.env.WEBEX_SCOPE,
-    authCodeOnly: options.authCodeOnly
+    roles: options.roles || [],
```

Comment:
Could you add these options to the JSDoc for `CreateUserOptions` defined on line 117?

Reply:
Updated - though you already seem to have some documentation for _ orgId_ which looked incorrect before.

Reply:
The `orgID` is in the documentation of the `TestUserObject` that comes back from test user creation.